lib/sufia/configuration.rb

```diff
@@ -7,6 +7,11 @@ def curation_concerns_config
     delegate(*(CurationConcerns.config.methods - Object.methods),
              to: :curation_concerns_config)
 
+    attr_writer :banner_image
+    def banner_image
+      @banner_image ||= "https://cloud.githubusercontent.com/assets/92044/18370978/88ecac20-75f6-11e6-8399-6536640ef695.jpg"
```

Comment:
Not hard, but why? This just bloats the gem and we expect that most users are going to set their own image anyway.